app/views/unattended/provisioning_templates/snippet/pxegrub2_chainload.erb

```diff
@@ -4,26 +4,12 @@ name: pxegrub2_chainload
 model: ProvisioningTemplate
 snippet: true
 description: |
-  In Foreman's typical PXE workflow, managed hosts are configured to always boot from network and inventory build flag dictates if they should boot into installer (build is on) or boot from local drive (build is off). This template is used to chainload from EFI ESP for systems which booted from network. It is not as straightforward as in BIOS and EFI boot file must be found on an ESP partition.
+  In Foreman's typical PXE workflow, managed hosts are configured to always boot from network and inventory build flag dictates if they should boot into installer (build is on) or boot from local drive (build is off). This template is used to load the GRUB2 configuration file or chainload from local EFI ESP for systems which booted from network.
```

Comment:
"This template tries to load first different GRUB2 configuration file from local ESP, then to chainload EFI binaries from ESP."